### ISSUE TITLE
non-array @properties of a custom type do not get interpreted

### ISSUE THREAD

**Issue by dolan** [tpitale/swagger_yard#27]
If I declare a property like this:

```ruby
@property [Array<FoobarClass>] foobars
```

The parsed response is correct, and includes an array of foobars. However...

```ruby
@property [FoobarClass] just_one_foobar
```

is rendered as a string with the sample text "FoobarClass" and any attributes of FoobarClass, even those marked with `@property` attributes themselves are ignored.

**Comment by tpitale** [comment]
From the looks of the specs/etc this just isn't supported yet. I'll add it post-spec2.

**Comment by tpitale** [comment]
Should be resolved by #29.
